tiny_dnn/layers/global_average_pooling_layer.h

```diff
@@ -109,7 +109,8 @@ class global_average_pooling_layer : public layer {
       core::OpKernelConstruction(layer::device(), &params_);
 
     layer::set_backend_type(backend_type);
-    if (backend_type == backend_t::internal || backend_type == backend_t::avx || backend_type == backend_t::nnpack) {
+    if (backend_type == backend_t::internal || backend_type == backend_t::avx ||
+        backend_type == backend_t::nnpack) {
```

Comment:
really, I think it can lead to future bugs. To prevent this you can use macros at testing time
https://github.com/tiny-dnn/tiny-dnn/blob/master/test/test_convolutional_layer.h#L212